hack/make/.build-deb/control

```diff
@@ -7,6 +7,7 @@ Vcs-Git: git://github.com/docker/docker.git
 Package: docker-engine
 Architecture: linux-any
 Depends: iptables, ${misc:Depends}, ${perl:Depends}, ${shlibs:Depends}
+Build-Depends: debhelper, dh-apparmor
```

Comment:
Yes, but we don't distribute source debs -- the debs are meant to be built
via "build-deb" only. :+1: